test/cli/cli-iexec-account.test.js

```diff
@@ -0,0 +1,244 @@
+        true,
+      );
+      expect(initialAccountBalance.add(bnAmount).eq(finalAccountBalance)).toBe(
+        true,
+      );
+    });
+
+    test('10 RLC', async () => {
+      const initialWalletBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} wallet show --raw`),
+        ).balance.nRLC,
+      );
+      const initialAccountBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} account show --raw`),
+        ).balance.stake,
+      );
+      const amount = '10';
+      const raw = await execAsync(
+        `${iexecPath} account deposit ${amount} RLC --raw`,
+      );
+      const res = JSON.parse(raw);
+      const bnAmount = new BN(amount).mul(new BN('1000000000'));
+      expect(res.ok).toBe(true);
+      expect(res.amount).toBe(bnAmount.toString());
+      expect(res.txHash).toBeDefined();
+      const finalWalletBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} wallet show --raw`),
+        ).balance.nRLC,
+      );
+      const finalAccountBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} account show --raw`),
+        ).balance.stake,
+      );
+      expect(initialWalletBalance.sub(bnAmount).eq(finalWalletBalance)).toBe(
+        true,
+      );
+      expect(initialAccountBalance.add(bnAmount).eq(finalAccountBalance)).toBe(
+        true,
+      );
+    });
+  });
+
+  describe('withdraw', () => {
+    beforeAll(async () => {
+      await runIExecCliRaw(`${iexecPath} account deposit 6 RLC`);
+    });
+
+    test('500 (nRLC)', async () => {
+      const initialWalletBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} wallet show --raw`),
+        ).balance.nRLC,
+      );
+      const initialAccountBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} account show --raw`),
+        ).balance.stake,
+      );
+      const amount = '500';
+      const raw = await execAsync(
+        `${iexecPath} account withdraw ${amount} --raw`,
+      );
+      const res = JSON.parse(raw);
+      expect(res.ok).toBe(true);
+      expect(res.amount).toBe(amount);
+      expect(res.txHash).toBeDefined();
+      const tx = await testChain.provider.getTransaction(res.txHash);
+      expect(tx).toBeDefined();
+      expect(tx.gasPrice.toString()).toBe('0');
+      const bnAmount = new BN(amount);
+      const finalWalletBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} wallet show --raw`),
+        ).balance.nRLC,
+      );
+      const finalAccountBalance = new BN(
+        JSON.parse(
+          await execAsync(`${iexecPath} account show --raw`),
+        ).balance.stake,
+      );
+      expect(initialWalletBalance.add(bnAmount).eq(finalWalletBalance)).toBe(
+        true,
+      );
+      expect(initialAccountBalance.sub(bnAmount).eq(finalAccountBalance)).toBe(
+        true,
+      );
+    });
+
+    test('5 RLC', async () => {
```

Comment:
it could be cool to test a random value instead

Reply:
the test didn't change, IMO this is out of this PR scope